src/test/munit/sftp-write-test-case-without-dir.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
+      xmlns:munit-tools="http://www.mulesoft.org/schema/mule/munit-tools"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:sftp="http://www.mulesoft.org/schema/mule/sftp"
+      xmlns:java="http://www.mulesoft.org/schema/mule/java"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+      http://www.mulesoft.org/schema/mule/test http://www.mulesoft.org/schema/mule/test/current/mule-test.xsd
+		http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+		http://www.mulesoft.org/schema/mule/munit-tools  http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd
+        http://www.mulesoft.org/schema/mule/sftp http://www.mulesoft.org/schema/mule/sftp/current/mule-sftp.xsd
+        http://www.mulesoft.org/schema/mule/java http://www.mulesoft.org/schema/mule/java/current/mule-java.xsd">
+
+    <munit:config name="sftp-write-test-case-without-dir.xml">
+        <munit:parameterizations>
+            <munit:parameterization name="config-docker">
+                <munit:parameters>
+                    <munit:parameter propertyName="config" value="config-docker"/>
+                </munit:parameters>
+            </munit:parameterization>
+        </munit:parameterizations>
+    </munit:config>
+
+    <munit:dynamic-port propertyName="sftp.server.port"/>
+
+    <munit:test name="sftp-write-on-home-directory">
+        <munit:behavior>
+            <java:invoke-static class="java.lang.System" method="setProperty(String,String)">
+                <java:args>
+                    #[{
+                    arg0: "mule.sftp.useHomeDirectory",
+                    arg1: "true"
```

Comment:
Please, could you add a test with this value in false (or as a default value) to validate the expected behavior in that case? (And remember to clean the properties in the after-test section)